lxd/storage/drivers/driver_zfs_volumes.go

```diff
@@ -1627,11 +1627,10 @@ func (d *zfs) SetVolumeQuota(vol Volume, size string, allowUnsafeResize bool, op
 	return nil
 }
 
-// GetVolumeDiskPath returns the location of a root disk block device.
-func (d *zfs) GetVolumeDiskPath(vol Volume) (string, error) {
+func (d *zfs) getVolumeDiskPathFromDataset(dataset string) (string, error) {
```

Comment:
Can you rebase the ` zfs: Add getVolumeDiskPathFromDataset ` commit into the main ZFS driver commit, as it makes it tricky to review the first commit if it changes later but depended on the later commit to function correctly?